src/components/Header/Heading.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import getElementType from '../../lib/getElementType';
+import './Heading.scss';
+
+const Heading = (props) => {
+  const {
+    className,
+    children,
+    size,
```

Comment:
> I think we should have some default sizes based on the `as` prop so the consumer only has to type `<Heading as="h1">Title</Heading>` to automatically get the size _we_ recommend.

@nathanyoung we can certainly do it that way. I was thinking it might be good to let users choose the element they want to render and size is as well. Based on context I've seen situations where it makes sense to use a particular kind of h tag but size it differently than others. E.G: a page where sections titles should have `h2` but the design might call for a smaller/larger size. 

Would like to get your thoughts on this.

Reply:
Yes, so i'd expect something like this:

```
<Heading as="h2">default size</Heading> // sets the size to XL by default

<Heading as="h2" size="sm">override size</Heading> // sets the size to SM
```

Reply:
Perfect that was my exact thinking I think I misunderstood your point about defaulting.